### DISCUSSION TITLE
Audio Sources Showing on All Devices

### DISCUSSION THREAD

**Discussion by cheuer** [File-New-Project/EarTrumpet#663] [Ideas]
There are certain audio sources (System Sounds and Steam notably) that apparently like to show on every single audio device, even though they only output to the default. Can these be excluded from all of the non-default audio devices so the list doesn't get like this?

![image](https://user-images.githubusercontent.com/1103172/41692250-6503d632-74bc-11e8-821f-2836f19242b4.png)

Edited to add: I'm on Windows 10 Version 1803 (OS Build 17134.112)

**Comment by riverar** [comment]
Hm we'll talk about this with the group and get back to you.

Few questions:
* What is that S app at the bottom? Just curious if there are icon retrieval improvements we can make.
* Are you really actively using that many devices? (You can disable them in Windows if not.)
* Would you expect to still see the device in this expanded view, just with no apps? Or would you expect us to hide it?

**Reply by PINPAL** [reply]
It would definetly be preferable if we could get the option to hide "System Sounds" from everything except the default output device just like in the Windows Volume Mixer.

For example I have some virtual outputs setup so I can split the audio in other applications such as OBS.
In this case, system sounds only ever come out of the default playback device however they show up under each possible output in EarTrumpet.
![image](https://user-images.githubusercontent.com/18197217/110867106-26209a00-82be-11eb-88d3-a8878c435f14.png)

**Comment by cheuer** [comment]
The S is the output from [VB-Audio virtual cable](https://www.vb-audio.com/Cable/index.htm) B. Not sure why, but it changes to a C when I right-click it:

![image](https://user-images.githubusercontent.com/1103172/41741149-82d860e2-7557-11e8-8477-793c8771360f.png)

Here's the icon that shows in the Legacy volume mixer:

![image](https://user-images.githubusercontent.com/1103172/41741064-401693aa-7557-11e8-8a57-91b4437d0b01.png)

I don't use them every day (so normally I leave this collapsed) but I do use them to separate/mix audio sources when streaming.

I could live with either way, it's useless when there's nothing playing on it, but if it just showed the header and some blank space it wouldn't be bad either.

It may also be worth noting, in the legacy volume mixer the system sounds only show on the default audio device, but the other ones (discord, steam etc) show on all of them, even though they don't play anything to them.

**Comment by drgruney** [comment]
I would like to second this. I am currently using a LOT of sound devices. I'm working on disabling in Windows the ones I never use, but System Sounds being on everything is a bit much.

![image](https://user-images.githubusercontent.com/24481663/82158969-e1b89f00-9850-11ea-9aba-ea76728b41f3.png)

**Comment by rp1231** [comment]
I opened up a similar topic in the issues section a few days back.

https://github.com/File-New-Project/EarTrumpet/issues/772

Just posting it here to consolidate things as there has been similar request on that thread as well.

Although the request to hide unwanted sliders is the same,
The request mostly pertains to audiodg.exe showing up in the volume sliders list.

**Comment by HaArD2000** [comment]
I also posted a similar thread...   [774](https://github.com/File-New-Project/EarTrumpet/discussions/774)

**Comment by jlitz** [comment]
never solved?

**Reply by HaArD2000** [reply]
No
